docs/rules/prefer-object-literal.md

```diff
@@ -1,6 +1,11 @@
 # prefer-object-literal
 
-Object literal syntax, which initializes an object's properties inside the object declaration is cleaner and clearer than the alternative: creating an empty object, and then giving it properites one by one.
+Object literal syntax, which initializes an object's properties inside the object declaration is cleaner and clearer than the alternative: creating an empty object, and then giving it properties one by one.
+
+An issue is raised when the following pattern is met:
+
+* An empty object is created.
```

Comment:
Add this to RSPEC

Reply:
Done [RSPEC-2428](https://jira.sonarsource.com/browse/RSPEC-2428)